lib/key/info.js

```diff
@@ -27,6 +27,10 @@ const packetInfo = (packet, key) => {
     return createPacketInfo(packet, key);
 };
 
+const getSubkeysFingerprints = (key) => {
```

Comment:
You can remove the if via destructuring -> if key can't be null.
```suggestion
const getSubkeysFingerprints = ({ subKeys = [] } = {}) => {
```

Reply:
You're right, I updated the code. The key cannot be `null` at this point in the code. If it's the case, an error is thrown in the `getKeys` function.